src/components/Proposals/DisputableStatusLabel.js

```diff
@@ -22,47 +21,48 @@ import {
   VOTE_STATUS_PAUSED,
   VOTE_STATUS_REJECTED,
 } from './disputable-vote-statuses'
+import disputed from '../../assets/disputed.svg'
 
 function getAttributes(status, theme) {
   const attributes = {
     [VOTE_STATUS_ACTIVE]: {
       label: 'Scheduled',
-      Icon: IconClock,
+      icon: <IconClock size="small" />,
     },
     [VOTE_STATUS_ACCEPTED]: {
       background: theme.surface,
       label: 'Passed',
-      Icon: IconCheck,
+      icon: <IconCheck size="small" />,
       color: theme.positive,
     },
     [VOTE_STATUS_CANCELLED]: {
       background: theme.surfaceUnder,
       label: 'Cancelled',
-      Icon: IconClose,
+      icon: <IconClose size="small" />,
```

Comment:
If we make the above change to the icon we won't have to update this code, it should just slot into the previous implementation.